### ISSUE TITLE
Add Navigation Bar

### ISSUE THREAD

**Issue by Nayanbatman** [Multiverse-of-Projects/NewsAI#79]
Add Navigation Bar . It will enhance the overall visual harmony and consistency of design .

**Comment by Nayanbatman** [comment]
please assign this issue to me under gssoc-extd and hacktoberfest

**Comment by Devasy23** [comment]
@Nayanbatman can you please elaborate the changes you want to make, and maybe a wireframe or some annotated screenshot for better explanation

**Comment by Nayanbatman** [comment]
@Devasy23 Yeah sure . If you add navigation bar on section " News AI Open to all Dashboard " It will look very good in the eyes of the user .

**Comment by Devasy23** [comment]
Go ahead and add that, but there aren't many tabs so what would you add to the navbar?
